samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -7,20 +7,54 @@ Bluetooth: Mesh light switch
    :local:
    :depth: 2
 
-The Bluetooth mesh light switch sample demonstrates how to set up a basic mesh client model application, and control LEDs with the Bluetooth mesh using the :ref:`bt_mesh_onoff_readme`.
+The :ref:`ug_bt_mesh` light switch sample can be used to change the state of light sources on other devices within the same mesh network.
+It also demonstrates how to use Bluetooth mesh models by using the Generic OnOff model in an application.
+
+Use the light switch sample with the :ref:`bluetooth_mesh_light` sample to demonstrate its function in a basic Bluetooth mesh network.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833
+
+You need at least two development kits:
+
+* One development kit where you program this sample application (the client)
+* One (or more) development kit(s) where you program the :ref:`bluetooth_mesh_light` sample application (the server(s)), and configure according to the mesh light sample's :ref:`testing guide <bluetooth_mesh_light_testing>`
+
+For provisioning and configuring of the mesh model instances, the sample requires a smartphone with Nordic Semiconductor's nRF Mesh mobile app installed in one of the following versions:
+
+* `nRF Mesh mobile app for Android`_
+* `nRF Mesh mobile app for iOS`_
 
 Overview
 ********
 
-This sample is split into two source files:
+The Bluetooth mesh light switch sample demonstrates how to set up a basic mesh client model application, and control LEDs with the Bluetooth mesh using the :ref:`bt_mesh_onoff_readme`.
+To display any functionality, the sample must be paired with a device with the :ref:`bluetooth_mesh_light` sample running in the same mesh network.
 
-* A :file:`main.c` file to handle initialization.
-* One additional file for handling mesh models, :file:`model_handler.c`.
+In both samples, devices are nodes with a provisionee role in a mesh network.
+Provisioning is performed in the `nRF Mesh mobile app`_.
```

Comment:
```suggestion
Provisioning is performed with the `nRF Mesh mobile app`_.
```

Reply:
Actually, looking at it now, I feel neither "in" nor "with" are quite a match. How about
- Provisioning is performed using the ...
or 
- Provisioning is performed from the ...